charts/pihole/templates/deployment.yaml

```
@@ -150,14 +150,10 @@ spec:
           - name: 'DNS2'
```

Do we need the quotes `'` around the env variable names? I'm a little confused, because I just recognized there are some  variables with quotes and some without. If we don't need them, maybe we can get rid of them entirely.

---

I prefer using them, but in this case they are indeed not needed.
When using special characters it might be best to keep them in.
```suggestion
          - name: DNS2
```

---

Thank you!